### ISSUE TITLE
Password for client certificates

### ISSUE THREAD

**Issue by nedo85** [sevcsik/client-certificate-demo#2]
Hi, there seems to be a password set for both the client certificates. I can't import the certificates without a password. Leaving the password field empty doesn't work. (MacOS + Chrome)

**Comment by TTTTMMMM** [comment]
Yeah -- on Mac keychain, it wants a passphrase, and when you leave it blank, you get an import error.
